.github/workflows/ci.yml

```diff
@@ -150,18 +150,18 @@ jobs:
 
 
     - name: Cache Zig packages
-      uses: actions/cache@v3
+      uses: actions/cache@v4
       with:
         path: ~/.cache/zig
-        key: ${{ runner.os }}-zig-packages-${{ hashFiles('build.zig.zon') }}
+        key: ${{ runner.os }}-zig-packages-${{ github.sha }}
```

Comment:
this effectively removes caching right?

Reply:
ugh, turns out it's a github runner bug. I think we can stay with this for now until it is fixed.

https://github.com/actions/runner/issues/4134